content/posts/newsletter/2020-08-06.md

```diff
@@ -0,0 +1,200 @@
+
+Anurodh Pokharel released [Monal 4.7 for iOS and macOS](https://github.com/anurodhp/Monal/releases/tag/Build_iOS_636). Despite the small version number update, this is a large upgrade. Besides the usual UI fixes and stability improvements you'll find: a new in chat title bar (as seen below), support for [XEP-0319](https://xmpp.org/extensions/xep-0319.html) (Last User Interaction in Presence) so you know when your contact was last seen online, support for [XEP-0085](https://xmpp.org/extensions/xep-0085.html) (Chat State Notifications) for those useful typing notifications and support for [XEP-0191](https://xmpp.org/extensions/xep-0191.html) (Blocking Command) for when a spammer comes along.
+
+![Monal navigation bar](/images/newsletter/july2020/Monal_navigation.png "Monal navigation bar") 
+
+The [git repository of Pix-Art Messenger](https://codeberg.org/kriztan/Pix-Art-Messenger/src/branch/master/CHANGELOG.md) has been moved from Github to Codeberg. This migration process is not finished yet. Until then, both repositories will run in parallel.
+Concerning the software itself, Pix-Art Messenger versions 2.5.1 and 2.5.2 have been released, highlighting pinned chats in color, improving the video quality, and dynamically optimizing file compression.
+
+Kontalk for Android has released a [new version (4.4.0)](https://github.com/kontalk/androidclient/releases)! Apart from bugfixes and improvements it has now a dark theme, supports sharing to groups, exporting media to storage and deleting media with messages. The settings have been revamped and Android 10 is now supported.
+
+The Android client aTalk released several patches for their 2.3.0 release with [bugfixes and improvements](https://github.com/cmeng-git/atalk-android/blob/master/aTalk/ReleaseNotes.txt).
+
+### Servers
+
+[MongooseIM 3.7.1 has been released!](https://github.com/esl/MongooseIM/releases/tag/3.7.1)
+This one is built on top of the 3.7.0 May release, which introduced support for channel binding to prevent [replay attacks](https://en.wikipedia.org/wiki/Replay_attack)
+for all methods of [SCRAM](https://en.wikipedia.org/wiki/Salted_Challenge_Response_Authentication_Mechanism "Salted Challenge Response Authentication Mechanism"),  a family of modern, password-based challenge–response authentication mechanisms providing authentication of a user to a server. This previous version also provided a range of new SCRAM authentication methods based on different flavors of the [SHA](https://en.wikipedia.org/wiki/Secure_Hash_Algorithms "Secure Hash Algorithms") cryptographic hash functions, the ability to retract messages as specified by [XEP-0424](https://xmpp.org/extensions/xep-0424.html), and support for [Proxy Protocol](https://www.haproxy.org/download/1.8/doc/proxy-protocol.txt) which safely transport connection information such as a client's address across multiple layers of NAT or TCP proxies. Version 3.7.1 improves on that by speeding up all the SCRAM methods significantly!
```

Comment:
```suggestion
for all methods of [SCRAM](https://en.wikipedia.org/wiki/Salted_Challenge_Response_Authentication_Mechanism "Salted Challenge Response Authentication Mechanism"),  a family of modern, password-based challenge–response authentication mechanisms providing authentication of a user to a server. This previous version also provided a range of new SCRAM authentication methods based on different flavors of the [SHA](https://en.wikipedia.org/wiki/Secure_Hash_Algorithms "Secure Hash Algorithms") cryptographic hash functions, the ability to retract messages as specified by [XEP-0424](https://xmpp.org/extensions/xep-0424.html), and support for [Proxy Protocol](https://www.haproxy.org/download/1.8/doc/proxy-protocol.txt) which safely transports connection information such as a client's address across multiple layers of NAT or TCP proxies. Version 3.7.1 improves on that by speeding up all the SCRAM methods significantly!
```

Reply:
This just changes the verb form which was introduced after my commit: "[Proxy protocol] which safely transport" -> "which safely transports".

Proxy Protocol was introduced in 3.7.0, which we missed the May newsletter with, hence a highlight of 3.7.0 features with the 3.7.1 announcement. Here are the 3.7.0 release notes: https://github.com/esl/MongooseIM/releases/tag/3.7.0

Reply:
Thanks! 6387ddd1f43a2de761fcee83dabffe8b28af93a5